### ISSUE TITLE
Get clean HDMI output in 1080 on an SLT a58

### ISSUE THREAD

**Issue by ofunny** [ma1co/Sony-PMCA-RE#378]
I would like to reuse an old SLT a58 for streaming. So far I see that the default firmware does not offer clean HDMI output and in other tickets here people failed to use the memory key that works for the a5000 and above.

Did anybody succeed already getting clean HDMI in 1080p from an SLT a58?
That would be awesome.

**Comment by ofunny** [comment]
I meanwhile dumped the Backup.bin and parsed the file to get a list of available keys, atm I just wait on the delivery of a HDMI to micro HDMI cable and than I will perform a trial error search for the clean HDMI key on the SLT a58 according to this approach https://github.com/ma1co/Sony-PMCA-RE/issues/313 – if someone already has the key or knows a better way to find the right key, please let me know.

**Comment by AbsusRex** [comment]
Did you ever find a solution for this or made any progress? If not I might pick this up, as soon as I find the time

In the meantime for everyone that stumbles upon this wanting to use the SLT-A58 as a webcam or similar: Putting the camera in movie mode, opening the ISO selection and cropping the area (with OBS) is the best workaround I've found so far.

**Comment by ofunny** [comment]
All I ever managed to get from the SLT-A58 was a resolution way below 1080p, not really usable. I managed to remove the auto focus indicator, however it came at the cost of losing auto focus at all. I could not find a key to get real clear HDMI. Not saying it's not possible but at least I was not able to find a working key.

**Comment by Ddupree96** [comment]
Hello, has there been any luck finding these keys? I have been scouring through my camera's firmware to find some sort of key that resembles HDMI out but no luck yet. Happy to see that there has been some recent activity regarding this.

**Comment by eldarbr** [comment]
Hello ofunny @ofunny, could you please provide me with your approach to hide the focusing indicator? I have an a65 (it is similar to a58).

**Comment by ofunny** [comment]
> Hello ofunny @ofunny, could you please provide me with your approach to hide the focusing indicator? I have an a65 (it is similar to a58).

I whish I could, but I did that a single time, half a year ago. All I remember is, that I connected via either service or backup shell, dumped the file (as mentioned in other Git conversations) than exported and filtered all the keys, I guess I made a script to speed that up and than just tried all possible combinations. 

Sadly, I did not keep any files of that test, but than again, I could not find anything remotely usable for the SLT-58 and the output was cropped, no full hd.
